### ISSUE TITLE
Configuration for google-spanner

### ISSUE THREAD

**Issue by stefy63** [typeorm/typeorm#9036]
Hi all, 
I have don't found documentation to implement google-spanner in my project. 
Please add this configuration and usage example. 
Thanks

**Comment by AlexMesser** [comment]
https://typeorm.io/#installation - Google Cloud Spanner section

**Comment by mprotsak** [comment]
Hi, I've tried 
```
npm install typeorm -g
typeorm init --name MyProject --database spanner
npm i
npm start
```
and also export GOOGLE_APPLICATION_CREDENTIALS

But unfortunately, it didn't work for me with projectId, instanceId and databaseId values that I used to connect to spanner with @google-cloud/spanner. I haven't found any examples of usage typeorm for spanner in github and I don't know if it is a bug or I am doing smth wrong.

My error after starting an app is 
`/projects/hg/MyProject/node_modules/@google-cloud/spanner/build/src/index.js:468
            throw new google_gax_1.GoogleError('A name is required to access an Instance object.');
                  ^
GoogleError: A name is required to access an Instance object.
    at Spanner.instance`
